spark-iotdb-connector/src/main/scala/org/apache/iotdb/spark/db/Converter.scala

```diff
@@ -65,7 +65,11 @@ object Converter {
       }
 
       val colCount = resultSetMetaData.getColumnCount
-      for (i <- 2 to colCount) {
+      var startIndex = 2
+      if (options.sql.toLowerCase.contains("count")) {
```

Comment:
All aggregation function should be filtered. A better way is check whether the first index column is "time".

Reply:
Ok, got this.